css/index.css

```diff
@@ -70,16 +68,16 @@
      transition: 0.3s ease;
      }
    
-     menu li {
+     .menu li {
      padding: 5px 14px;
      }
      
-     <!--Checkbox-->
-     input [type=checkbox]:checked ~ .menu { 
-     display: inline-block;
+     /*CHECKBOX*/
+     input[type=checkbox] {
+      display:none;
      }
    
-     <!--Hamburger Menu-->
+     /*Hamburger Menu*/
```

Comment:
On idiom for the 3-bar hamburger is for the menu to expand when you click on it, but nothing seems to be happening here.